packages/core-mobile/app/screens/bridge/BridgeTokenSelector.tsx

```diff
@@ -68,7 +64,11 @@ function BridgeTokenSelector({
         rightComponentVerticalAlignment={'center'}
         rightComponent={
           <Text variant="body1" style={{ marginLeft: 12 }}>
-            {formatTokenAmount(token.balance || BIG_ZERO, 6)} {token.symbol}
+            {formatTokenAmount(
+              bigintToBig(token.balance || 0n, denomination),
+              6
```

Comment:
I think we shouldn't do 0n here and instead do the `-`

Reply:
fixed.